src/master-playlist-controller.js

```diff
@@ -62,7 +62,8 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     this.mode_ = mode;
     this.useCueTags_ = useCueTags;
     if (this.useCueTags_) {
-      this.cueTagsTrack_ = this.tech_.addTextTrack('metadata', 'hls-segment-metadata');
+      this.cueTagsTrack_ = this.tech_.addTextTrack('metadata',
+        'hls-segment-metadata');
```

Comment:
We should rename this TextTrack to 'ad-cues' as it no longer contains 'hls-segment-metdata'.